activiti-spring-security-policies/src/test/java/org/activiti/spring/security/policies/RuntimeTestConfiguraiton.java

```diff
@@ -17,28 +17,27 @@
 public class RuntimeTestConfiguraiton {
 
     @Bean
-    public UserDetailsService myUserDetailsService() {
-        InMemoryUserDetailsManager inMemoryUserDetailsManager = new InMemoryUserDetailsManager();
+    public UserDetails myUserDetailsService() {
+        ExtendedInMemoryUserDetailsManager extendedInMemoryUserDetailsManager = new ExtendedInMemoryUserDetailsManager();
 
         List<GrantedAuthority> salaboyAuthorities = new ArrayList<>();
         salaboyAuthorities.add(new SimpleGrantedAuthority("ROLE_ACTIVITI_USER"));
         salaboyAuthorities.add(new SimpleGrantedAuthority("GROUP_activitiTeam"));
         salaboyAuthorities.add(new SimpleGrantedAuthority("GROUP_developers"));
 
-        inMemoryUserDetailsManager.createUser(new User("salaboy", "password", salaboyAuthorities));
+        extendedInMemoryUserDetailsManager.createUser(new User("salaboy", "password", salaboyAuthorities));
 
         List<GrantedAuthority> adminAuthorities = new ArrayList<>();
         adminAuthorities.add(new SimpleGrantedAuthority("ROLE_ACTIVITI_ADMIN"));
 
-        inMemoryUserDetailsManager.createUser(new User("admin", "password", adminAuthorities));
+        extendedInMemoryUserDetailsManager.createUser(new User("admin", "password", adminAuthorities));
 
         List<GrantedAuthority> garthAuthorities = new ArrayList<>();
         garthAuthorities.add(new SimpleGrantedAuthority("ROLE_ACTIVITI_USER"));
         garthAuthorities.add(new SimpleGrantedAuthority("GROUP_doctor"));
 
-        inMemoryUserDetailsManager.createUser(new User("garth", "password", garthAuthorities));
+        extendedInMemoryUserDetailsManager.createUser(new User("garth", "password", garthAuthorities));
 
-        return inMemoryUserDetailsManager;
+        return (UserDetails) extendedInMemoryUserDetailsManager;
```

Comment:
@pancudaniel7 I guess that we are not testing this one. because I guess that this will break in runtime.. We need to use UserDetailsService